dashboard/app/models/user.rb

```diff
@@ -194,11 +194,12 @@ def school_info_optional?
 
   belongs_to :invited_by, polymorphic: true
 
-  validate :admins_must_be_teachers
+  validate :admins_must_be_teachers_without_followeds
```

Comment:
nit: the english word "followees" would be better than the non-word "followeds" here. optional to change it in this PR, but this may be worth changing it some point considering that it wouldn't require a db migration.